client/src/components/ReviewTile.js

```diff
@@ -1,13 +1,50 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 
-const ReviewTile = ({ review }) => {
-  const { subject, body, rating } = review
+const ReviewTile = ({ review, castVote, currentUser }) => {
+  const { id, author, subject, body, rating, votes } = review
+  const { score, currentUserVote } = votes
+  const [voteState, setVoteState] = useState(null)
+
+  const handleButtonClick = async (event) => {
+    if (currentUser) {
+      const { currentTarget } = event
```

Comment:
i wouldnt destructure this. I think its better to just do event.currentTarget.value, unless you destructure inline on line 8